examples/tooltip-on-hover.js

```diff
@@ -0,0 +1,77 @@
+import Fill from '../src/ol/style/Fill.js';
+import GeoJSON from '../src/ol/format/GeoJSON.js';
+import Map from '../src/ol/Map.js';
+import Style from '../src/ol/style/Style.js';
+import VectorLayer from '../src/ol/layer/Vector.js';
+import VectorSource from '../src/ol/source/Vector.js';
+import View from '../src/ol/View.js';
+
+const style = new Style({
+  fill: new Fill({
+    color: '#eeeeee',
+  }),
+});
+
+const vector = new VectorLayer({
+  source: new VectorSource({
+    url: 'https://openlayers.org/data/vector/ecoregions.json',
+    format: new GeoJSON(),
+  }),
+  background: 'white',
+  style: function (feature) {
+    const color = feature.get('COLOR') || '#eeeeee';
+    style.getFill().setColor(color);
+    return style;
+  },
```

Comment:
```suggestion
  style: {
    'fill-color': ['string', ['get', 'COLOR'], '#eeeeee'],
  },
```

We're moving away from using the style and symbolizer constructors.  Can you update the example to use a style like the one above and remove the call to the `Style` constructor above?

Reply:
Thanks for quick review. Suggested changes applied.